pom.xml

```diff
@@ -684,6 +690,11 @@
             <name>JBoss Maven Repository</name>
             <url>http://repository.jboss.org/nexus/content/groups/public/</url>
         </repository>
+        <repository>
+            <id>lz4-maven-repository</id>
+            <name>LZ4 Maven Repository</name>
+            <url>http://repo1.maven.org/maven2/net/jpountz/lz4</url>
```

Comment:
for my information.. do we have to define the this repository explicitly.
I though it will get downloaded automatically from maven central.